### PR TITLE
Add ability to set a class or hide menu actions in List View

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
This addresses issue #306 

@dgutride @dtaylor113 @AparnaKarve

### CODE REVIEW

**Comment by dtaylor113** [comment]
Looks good.  Any unit test coverage for these new options?

**Comment by jeff-phillips-18** [comment]
@dtaylor113 Added unit tests for these options

File: src/views/listview/list-view-directive.js [hunk 1]
```diff
@@ -238,6 +242,18 @@
           },
         ];
 
+        $scope.getMenuClass = function (item) {
+          var menuClass = "";
+          if (item.name === "Jim Beam") {
```
Comment (dgutride) [inline src/views/listview/list-view-directive.js]: Maybe a slightly less risque name here?  Not sure if we have public standards for documentation or if I'm just being overly cautious...

**Comment by jeff-phillips-18** [comment]
Heh, that name has been used here since its inception. Do you think I should change it?

File: src/views/listview/list-view-directive.js (hunk 1 above)
Comment (jeff-phillips-18) [inline src/views/listview/list-view-directive.js]: I've changed the name to protect the innocent 👮

**Comment by dgutride** [comment]
I'm good with this.

**Comment by AparnaKarve** [comment]
Thanks for this PR @jeff-phillips-18 
I haven't tested this yet, but I'm A-OK with @dtaylor113 and @dgutride 's review 👍

**Comment by AparnaKarve** [comment]
Quick question: To be able to test these changes with SSUI, do I have to wait for the next version of angular-patternfly ? Currently SSUI uses `"angular-patternfly": "^3.11.0"`

**Comment by jeff-phillips-18** [comment]
Yes, angular patternfly should release 3.12.0 either today or very shortly and this PR looks to be included.

**Pull request merged**